examples/pico_display_2/pico_display_2_vector.cpp

```diff
@@ -51,7 +51,7 @@ int main() {
         pp_mat3_translate(&pos, 50, 50);
         pp_mat3_rotate(&pos, a);
         vector.draw(poly);
-        vector.text("Hello World", &pos);
+        vector.text("Hello World", 0, 0, &pos);
```

Comment:
Alternatively could make these optional arguments if you prefer - I had kind of forgotten that this was also exposed to C++.

Reply:
I think it would be good to make these optional, since it also bubbles up to Python. C isn't super helpful about that, though. I wonder if it makes sense to have them part of the text metrics, or if that's just too conceptually weird.

Sensible API design for this has been quite tricky!